cmd/kyma/alpha/cmd.go

```diff
@@ -0,0 +1,14 @@
+package alpha
+
+import (
+	"github.com/spf13/cobra"
+)
+
+//NewCmd creates a new alpha command
+func NewCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "alpha",
+		Short: "Executes the commands in alpha testing stage.",
```

Comment:
```suggestion
		Short: "Executes the commands in alpha testing stage.",
		Long:  "Use this command to execute commands in alpha testing stage.",
```
In other docs, there is always a description starting with "Use this command..." and on the preview the same text is shown twice:

<img width="400" alt="Screen Shot 2020-11-30 at 12 44 10" src="https://user-images.githubusercontent.com/49867570/100606818-48dc0380-330a-11eb-86b1-75f31ab10d88.png">

So I wanted to re-word it somehow.